actions/bump-version/action.yaml

```diff
@@ -10,7 +10,7 @@ runs:
   steps:
     - name: 'Get next version'
       shell: bash
-      run: echo "next_version=$(cz bump --get-next)" >> $GITHUB_ENV
+      run: echo "version=$(cz bump --get-next)" >> $GITHUB_ENV
```

Comment:
The name is arbitrary, but the specified output above and all the docs are using `version` instead of `next_version`.